Emergency Fixes, Readme / Doc Changes

mooleshacat opened the pull request:
Hey

I added a few config variables (so it uses stratum host for config generator)

I also forked node-stratum-pool and implemented a fix. You will need to fork the original and fix it or fork mine, or just change your package.json to install mine.

   * Fix for https://github.com/foxer666/node-stratum-pool/commit/a07031808059a76a1fc61db9b10099155df1b6bb
   * Fix for https://github.com/foxer666/node-stratum-pool/pull/23/commits/67eace3468a77f7a585303fe8a1a2081eeba4096

Will be pushing block explorer changes seperately.

~Shawn

leshacat commented:
Theres a bunch of other changes in here, readme, documentation.... May start working on docker temporarily or possibly go back to block explorer

leshacat commented:
You should probably also use a "development" branch as well so if you make changes and it breaks it then people doing git clone won't be affected until you merge

mooleshacat committed:
Update INSTALL.md

mooleshacat committed:
Update DOCKER.md

mooleshacat committed:
Update CHANGELOG.md

mooleshacat committed:
Merge pull request #44 from leshacat/development

Development

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #45 from leshacat/development

Development merge

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
added shit ton of coin configs from another fork

mooleshacat committed:
Merge branch 'development' of https://github.com/leshacat/BootNOMP into development

mooleshacat committed:
Merge pull request #46 from leshacat/charity-output

Charity output

mooleshacat committed:
Update INSTALL.md

mooleshacat committed:
Update INSTALL.md

mooleshacat committed:
Merge pull request #47 from leshacat/development

Development

mooleshacat committed:
Merge pull request #48 from leshacat/master

unknown merge

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #49 from leshacat/development

Development

mooleshacat committed:
Update INSTALL.md

mooleshacat committed:
Merge pull request #50 from leshacat/master

unknown merge

mooleshacat committed:
update

mooleshacat committed:
Docker & Docs/Install

mooleshacat committed:
Docker & Docs/Install

mooleshacat committed:
Merge pull request #51 from leshacat/development

 Docker & Docs/Install

mooleshacat committed:
Docker & Docs/Install

mooleshacat committed:
Merge pull request #52 from leshacat/development

Docker & Docs/Install

mooleshacat committed:
Docker & Docs/Install

mooleshacat committed:
Docker & Docs/Install

mooleshacat committed:
Docker & Docs/Install

mooleshacat committed:
Docker & Docs/Install (#53)

* Docker & Docs/Install

* Docker & Docs/Install

* Docker & Docs/Install

mooleshacat committed:
Docker & Docs/Install

mooleshacat committed:
Docker Updates (#54)

* Docker & Docs/Install

* Docker & Docs/Install

* Docker & Docs/Install

* Docker & Docs/Install

mooleshacat committed:
Update DOCKER.md

mooleshacat committed:
Merge branch 'master' into development

mooleshacat committed:
Docker (#55)

* Docker & Docs/Install

* Docker & Docs/Install

* Docker & Docs/Install

* Docker & Docs/Install

mooleshacat committed:
Update DOCKER.md

mooleshacat committed:
Update INSTALL.md

mooleshacat committed:
Update CHANGELOG.md

mooleshacat committed:
Update TODO.md

mooleshacat committed:
Update TODO.md

mooleshacat committed:
Update TODO.md

mooleshacat committed:
Update TODO.md

mooleshacat committed:
Update TODO.md

mooleshacat committed:
Update TODO.md

mooleshacat committed:
Update TODO.md

mooleshacat committed:
Merge branch 'master' into development

mooleshacat committed:
Development (#56)

* Docker & Docs/Install

* Docker & Docs/Install

* Docker & Docs/Install

* Docker & Docs/Install

mooleshacat committed:
Development (#57)

* Docker & Docs/Install

* Docker & Docs/Install

* Docker & Docs/Install

* Docker & Docs/Install

mooleshacat committed:
unknown merge (#58)

* Development (#56)

* Docker & Docs/Install

* Docker & Docs/Install

* Docker & Docs/Install

* Docker & Docs/Install

* Development (#57)

* Docker & Docs/Install

* Docker & Docs/Install

* Docker & Docs/Install

* Docker & Docs/Install

mooleshacat committed:
Development (#59)

* Docker & Docs/Install

* Docker & Docs/Install

* Docker & Docs/Install

* Docker & Docs/Install

* unknown merge (#58)

* Development (#56)

* Docker & Docs/Install

* Docker & Docs/Install

* Docker & Docs/Install

* Docker & Docs/Install

* Development (#57)

* Docker & Docs/Install

* Docker & Docs/Install

* Docker & Docs/Install

* Docker & Docs/Install

mooleshacat committed:
Merge pull request #60 from leshacat/development

Development

mooleshacat committed:
Merge pull request #61 from leshacat/master

Unknown

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #62 from leshacat/development

Merge pull request #61 from leshacat/master

mooleshacat committed:
Merge pull request #63 from leshacat/master

master->development

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #64 from leshacat/development

Development

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #65 from leshacat/development

Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #66 from leshacat/development

Documentation Change

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #67 from leshacat/master

Docs

mooleshacat committed:
Update TODO.md

Changed into a table :)

mooleshacat committed:
Update TODO.md

mooleshacat committed:
Update TODO.md

mooleshacat committed:
Merge pull request #68 from leshacat/development

Updated TODO.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Updated Credits

Added repository owners to top of credits

mooleshacat committed:
Merge pull request #69 from leshacat/development

Added repo owners to Credits

mooleshacat committed:
Adding screenshots folder

mooleshacat committed:
Adding Screenshots for README.md

Adding Screenshots for README.md

mooleshacat committed:
Delete placeholder.md

mooleshacat committed:
Update README.md

Added screenshots to README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #70 from leshacat/development

Adding Screenshots to README.md

mooleshacat committed:
Fixed Screenshots

Fixed Screenshots

mooleshacat committed:
Merge pull request #71 from leshacat/development

Fixed Screenshots

mooleshacat committed:
Fixed Screenshots in README.md

Fixed Screenshots in README.md

mooleshacat committed:
Merge pull request #72 from leshacat/development

Fixed Screenshots in README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #73 from leshacat/development

Development

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #74 from leshacat/master

misc merge

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #75 from leshacat/development

Development

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #76 from leshacat/master

master -> development

mooleshacat committed:
Merge pull request #77 from leshacat/development

Updated before starting development on patch

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #78 from leshacat/development

Development Merge

mooleshacat committed:
Merge pull request #79 from leshacat/development

Development Merge

mooleshacat committed:
Merge pull request #80 from leshacat/payment-processor-patch-1

Payment processor patch 1

mooleshacat committed:
Merge pull request #81 from leshacat/development

Development

mooleshacat committed:
Update README.md

mooleshacat committed:
Update Dockerfile

mooleshacat committed:
Update Dockerfile

mooleshacat committed:
Merge pull request #82 from leshacat/master

master -> dev

mooleshacat committed:
Update INSTALL.md

mooleshacat committed:
Update package.json

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #83 from leshacat/development

Name change BootNOMP -> EasyNOMP

mooleshacat committed:
added powered by link

mooleshacat committed:
Merge pull request #84 from leshacat/development

added powered by link

mooleshacat committed:
changed powered by link

mooleshacat committed:
Merge pull request #85 from leshacat/development

changed powered by link

mooleshacat committed:
changed fonsize in powered by link

mooleshacat committed:
Merge pull request #86 from leshacat/development

changed fonsize in powered by link

mooleshacat committed:
update

mooleshacat committed:
update

mooleshacat committed:
Update README.md

Updated LeshaCat statement :)

mooleshacat committed:
Update README.md

Added website links

mooleshacat committed:
Update README.md

Updated website links

mooleshacat committed:
Update README.md

Updated website links

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #87 from leshacat/website-menu

Website menu

mooleshacat committed:
Merge pull request #88 from leshacat/development

Development

mooleshacat committed:
Merge pull request #89 from leshacat/master

merge

mooleshacat committed:
Merge pull request #90 from leshacat/development

Merge pull request #89 from leshacat/master

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #93 from leshacat/master

master to development

mooleshacat committed:
Fix for "undefined" warning-msg

Fix for "undefined" warning-msg

mooleshacat committed:
Merge pull request #94 from leshacat/warning-msg-patch-1

Fix for "undefined" warning-msg

mooleshacat committed:
Merge pull request #95 from leshacat/development

Development

mooleshacat committed:
Fix for stale z-nomp redis data

Added redis-cli FLUSHALL to install instructions so if someone migrates from z-nomp the redis data is flushed.

mooleshacat committed:
Update INSTALL.md

Moved redis-cli FLUSHALL to it's own section with warning

mooleshacat committed:
Update INSTALL.md

Updated FLUSHALL instructions

mooleshacat committed:
Merge pull request #96 from leshacat/development

Development

mooleshacat committed:
Update INSTALL.md

mooleshacat committed:
Merge pull request #97 from leshacat/development

Update INSTALL.md

mooleshacat committed:
Update INSTALL.md

mooleshacat committed:
Merge pull request #98 from leshacat/development

Update INSTALL.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #99 from leshacat/development

Development

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #100 from leshacat/development

Update README.md

mooleshacat committed:
Fix title metatag undefined bug

mooleshacat committed:
Update index.html

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

Update to payment bug reference links

mooleshacat committed:
Update README.md

Update to payment bug reference links

mooleshacat committed:
Update README.md

Update to payment bug reference links

mooleshacat committed:
Update README.md

Update to payment bug reference links

mooleshacat committed:
Update README.md

https://github.com/foxer666/node-open-mining-portal/issues/145

mooleshacat committed:
Merge pull request #101 from leshacat/development

Fixes

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #102 from leshacat/development

Development

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #103 from leshacat/development

Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #104 from leshacat/development

Development

mooleshacat committed:
Update index.html

mooleshacat committed:
Merge pull request #105 from leshacat/development

Update index.html

mooleshacat committed:
Update index.html

mooleshacat committed:
Merge pull request #106 from leshacat/development

Update index.html

mooleshacat committed:
Updated Keywords & Description - undefined bug

mooleshacat committed:
Merge pull request #107 from leshacat/master

dev -> master

mooleshacat committed:
Merge pull request #108 from leshacat/development

Merge pull request #107 from leshacat/master

mooleshacat committed:
fixed warning msg

mooleshacat committed:
Merge branch 'master' of https://github.com/leshacat/EasyNOMP

mooleshacat committed:
Merge pull request #109 from leshacat/master

master -> dev

mooleshacat committed:
Merge pull request #110 from leshacat/development

Merge pull request #109 from leshacat/master

mooleshacat committed:
fixed footer

mooleshacat committed:
Merge branch 'master' of https://github.com/leshacat/EasyNOMP

mooleshacat committed:
Merge pull request #111 from leshacat/master

footer fix

mooleshacat committed:
fixed warningmsg border

mooleshacat committed:
Merge pull request #112 from leshacat/master

fixed warningmsg border

mooleshacat committed:
update

mooleshacat committed:
Merge pull request #113 from leshacat/master

update

mooleshacat committed:
update

mooleshacat committed:
Merge branch 'development' of https://github.com/leshacat/EasyNOMP into development

mooleshacat committed:
Merge pull request #114 from leshacat/development

Development

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #115 from leshacat/development

Development

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Merge pull request #116 from leshacat/master

master -> dev

mooleshacat committed:
Update README.md

mooleshacat committed:
Create install.sh

mooleshacat committed:
Merge pull request #117 from leshacat/development

Development

mooleshacat committed:
update

mooleshacat committed:
update

mooleshacat committed:
Merge pull request #118 from leshacat/development

Development

mooleshacat committed:
Updated install instructions: install.sh

mooleshacat committed:
Merge pull request #119 from leshacat/development

Updated install instructions: install.sh

mooleshacat committed:
Merge pull request #120 from leshacat/master

merge

mooleshacat committed:
install script update

mooleshacat committed:
Merge pull request #121 from leshacat/development

Development

mooleshacat committed:
install script update

mooleshacat committed:
Merge pull request #122 from leshacat/development

install script update

mooleshacat committed:
install script update

mooleshacat committed:
Merge pull request #123 from leshacat/development

install script update

mooleshacat committed:
Merge pull request #124 from leshacat/master

unknown

mooleshacat committed:
install script update

mooleshacat committed:
update todo

mooleshacat committed:
Merge branch 'development' of https://github.com/leshacat/EasyNOMP into development

mooleshacat committed:
Merge pull request #125 from leshacat/development

Development

mooleshacat committed:
update

mooleshacat committed:
update install.sh

mooleshacat committed:
Merge pull request #126 from leshacat/development

update install.sh

mooleshacat committed:
readme update - features

mooleshacat committed:
Merge pull request #127 from leshacat/development

readme update - features

mooleshacat committed:
readme update - features

mooleshacat committed:
Merge pull request #128 from leshacat/development

readme update - features

mooleshacat committed:
readme update - features

mooleshacat committed:
readme update - features

mooleshacat committed:
readme update - features

mooleshacat committed:
readme update - features

mooleshacat committed:
readme update - features

mooleshacat committed:
readme update - features

mooleshacat committed:
readme update - features

mooleshacat committed:
readme update - features

mooleshacat committed:
readme update - features

mooleshacat committed:
todo update

mooleshacat committed:
Merge pull request #129 from leshacat/development

Development

mooleshacat committed:
readme update - support

mooleshacat committed:
Merge pull request #130 from leshacat/development

readme update - support

mooleshacat committed:
readme update - support

mooleshacat committed:
readme update - support

mooleshacat committed:
Merge pull request #131 from leshacat/development

Development

mooleshacat committed:
Update install.sh

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

mooleshacat committed:
Update README.md

leshacat commented:
I am going to close this and reopen my PR's with cherry-picked commits so your documentation does not change :)
